audio-worklet/basic/audio-worklet-node-options/index.html

```diff
@@ -93,13 +93,12 @@
         }
       }
     }
-
+    
     document.querySelector('#toogle').addEventListener('click', demoFunction);
-    document.querySelector('#demo-input-type').addEventListener('change', demoFunction);
-    document.querySelector('#demo-input-frequency').addEventListener('change', demoFunction);
-
-
-
+    document.querySelector('#demo-input-type').addEventListener('change', 
```

Comment:
https://google.github.io/styleguide/jsguide.html#formatting-indent

I would actually prefer:
```js
document.querySelector('#demo-input-type')
        .addEventListener('change', demoFunction);
```

Reply:
Sure!! That looks better